configuration/download_cache.rst

```diff
@@ -0,0 +1,92 @@
+
+.. warning::
+
+    This is an **advanced** feature, use it only if you fully understand this document. Do not use it if you are mixing api V1 (without revisions)
+    and api V2 (with revisions) clients.
+
+
+
+Conan implements a shared download cache that can be used to reduce the time needed to populate the Conan package cache
+with commands like :command:`install`, :command:`create`.
+
+This cache is purely an optimization mechanism. It is completely different to the :ref:`Conan package cache <custom_cache>`, (typically the ``<userhome>/.conan`` folder).
+It is not related to the ``short_paths`` mechanism for long path in Windows, nor to the ``short_paths`` cache folder. The cache will contain a copy
+of the artifacts, it is not a new location of files. Those files will still be copied to the Conan package cache, which will not change anything,
+its behavior, layout or location of any file.
+
+This cache (whose path can be configured in the *conan.conf* file) will store the following items:
+
+- All files that are downloaded from a Conan server (conan_server, Artifactory), both in the api V1 (without revisions) and V2 (with revisions).
+  This includes files like *conanfile.py*, but also the zipped artifacts like *conan_package.tgz* or *conan_sources.tgz*.
+- The downloads done by users with the ``tools.download()`` or ``tools.get()`` helpers, as long as they provide a checksum (md5, sha1, etc.). If
+  a checksum is not provided, even if the download cache is enabled, the download will be always executed and the files will not be cached.
+
+The cache uses the URL of the download, appending the checksum when provided. In the api V2, it is not necessary to append the checksum, because
+the URL itself already encodes the recipe revision and/or the package revisions, which are already checksums of the recipe and package respectively.
+
+.. warning::
+
+    The download cache will not be able to correctly cache artifacts with revisions enabled if those artifacts are created and uploaded repeatedly
+    in a client without revisions, because that will keep overwriting the revision "0".
+
+Activating/deactivating the download cache
+------------------------------------------
+
+The download cache is activated and configured in the :ref:`conan_conf` like this:
+
+.. code-block:: text
+
+    [storage]
+    download_cache=/path/to/my/cache
+
+It can be defined from the command like as well:
+
+.. code-block:: bash
+
+    $ conan config set storage.download_cache="/path/to/my/cache"
+    # Display it
+    $ conan config get storage.download_cache
+
+
+And, as the *conan.conf* is part of the configuration, you can also put a common *conan.conf* file in a git repo or zip file and use
+the :ref:`conan_config_install` command to automatically install it in Conan clients.
+
+
+To deactivate the download cache, you can remove the entry ``download_cache`` from the *conan.conf* file directly editing it, or with the command:
```

Comment:
```suggestion
To deactivate the download cache, you can remove the entry ``download_cache`` from the *conan.conf* with the command:
```

I wouldn't promote editing that file directly (people already know they can do that)